Spigot-Server-Patches/0168-Disable-ticking-of-snow-blocks.patch

```diff
@@ -0,0 +1,43 @@
+From 256760656fdfaaac5225dc5f7c719c4bb09fbc4b Mon Sep 17 00:00:00 2001
+From: killme <killme-git@ibts.me>
+Date: Tue, 30 Aug 2016 16:39:48 +0200
+Subject: [PATCH] Disable ticking of snow blocks
+
+
+diff --git a/src/main/java/net/minecraft/server/BlockSnowBlock.java b/src/main/java/net/minecraft/server/BlockSnowBlock.java
+index 88865aa..756c6ca 100644
+--- a/src/main/java/net/minecraft/server/BlockSnowBlock.java
++++ b/src/main/java/net/minecraft/server/BlockSnowBlock.java
+@@ -6,7 +6,7 @@ public class BlockSnowBlock extends Block {
+ 
+     protected BlockSnowBlock() {
+         super(Material.SNOW_BLOCK);
+-        this.a(true);
++        // this.a(true); // Paper - snow blocks don't need to tick
+         this.a(CreativeModeTab.b);
+     }
+ 
+@@ -18,11 +18,13 @@ public class BlockSnowBlock extends Block {
+         return 4;
+     }
+ 
+-    public void b(World world, BlockPosition blockposition, IBlockData iblockdata, Random random) {
+-        if (world.b(EnumSkyBlock.BLOCK, blockposition) > 11) {
+-            this.b(world, blockposition, world.getType(blockposition), 0);
+-            world.setAir(blockposition);
+-        }
+-
+-    }
++// Paper start - snow blocks don't need to tick
```

Comment:
Not completely sure about the first part. I took a quick look over the vanilla code and it does not seems to be completely respected everywhere.

And sure, will do.